src/include/storage.h

```diff
@@ -63,6 +69,26 @@ pgmoneta_storage_create_s3(void);
 struct workflow*
 pgmoneta_storage_create_azure(void);
 
+/**
+ * Open WAL shipping file in remote ssh server
+ * @param srv The server index
+ * @param filename WAL file name
+ * @param segsize WAL segment size
+ * @return sftp_file WAL streaming file
+ */
+sftp_file
```

Comment:
Use an out parameter instead, and `int` as the return type with the status of the operation